lib/Migration/Version2060Date20200302132145.php

```diff
@@ -11,23 +11,69 @@
 
 class Version2060Date20200302132145 extends SimpleMigrationStep {
 
-	/**
-	 * @param IOutput $output
-	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
-	 * @param array $options
-	 * @return null|ISchemaWrapper
-	 */
-	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options) {
-		/** @var ISchemaWrapper $schema */
-		$schema = $schemaClosure();
-
-		$table = $schema->getTable('richdocuments_wopi');
-		$table->addColumn('share', 'string', [
-			'notnull' => false,
-			'length' => 64
-		]);
-
-		return $schema;
-	}
-
-}
+    /**
+     * @param IOutput $output
+     * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+     * @param array $options
+     * @return null|ISchemaWrapper
+     */
+    public function changeSchema(IOutput $output, Closure $schemaClosure, array $options) {
+        /** @var ISchemaWrapper $schema */
+        $schema = $schemaClosure();
+
+        $table = $schema->getTable('richdocuments_wopi');
+
+        if(!$table->hasColumn('template_id')) {
```

Comment:
Could you create a new migration for all of those checks, so we make sure they are also executed if someone has already updated but not encountered the issue yet?